src/model.py

```diff
@@ -153,15 +167,9 @@ def init_engine(self):
             AsyncEngineArgs(**self.vllm_engine_config)
         )
 
-        # If TRITON_ENABLE_METRICS<_CPU/_GPU> build flag is enabled.
-        if self.args["metrics_mode"] in ["all", "cpu", "gpu"]:
-            # Create vLLM custom Metrics
-            labels = {
-                "model": self.args["model_name"],
-                "version": self.args["model_version"],
-            }
-            logger = VllmStatLogger(labels=labels)
-            self.llm_engine.add_logger("triton", logger)
+        # Add vLLM custom metrics
+        if not self.metrics:
```

Comment:
I think this should be inverted, `if self.metrics: add_logger`

Reply:
Yeah. Found that after testing.